src/python/pants/backend/codegen/tasks/protobuf_gen.py

```diff
@@ -63,6 +63,10 @@ def register_options(cls, register):
              help='Dependencies to bootstrap this task for generating java code.  When changing '
                   'this parameter you may also need to update --version.',
              default=['3rdparty:protobuf-java'])
+    register('--import-from-root', type=bool, advanced=True,
+             help='If set, add the buildroot to the path protoc searches for imports. '
+                  'This facilitates import paths in .proto files that are relative to the '
+                  'build root, as recommended by the protoc documentation.')
```

Comment:
The second sentence feels a little awkward, maybe something like

> This enables using import paths relative to the build root in .proto files, as recommended by the protoc documentation.

would be better